specs/latest/2.0/index.html

```diff
@@ -1869,7 +1869,7 @@ <h2 class="no-toc">Table of contents</h2>
         </dt>
         <dd>
           <p>If no WebGLBuffer is bound to the <code>PIXEL_PACK_BUFFER</code> target, generates an <code>INVALID_OPERATION</code> error.</p>
-          <p><em>offset</em> is the byte offset into the WebGLBuffer's data store; generates an <code>INVALID_VALUE</code> error if it's less than 0.</p>
+          <p><em>offset</em> is the byte offset into the WebGLBuffer's data store; generates an <code>INVALID_VALUE</code> error if it's less than 0. If <em>offset</em> passes the end of the pixel pack buffer, or the pixel pack buffer is not large enough to store all pixels, generates an <code>INVALID_OPERATION</code>.</p>
```

Comment:
I thought about this again. The other readPixels API would generate INVALID_ENUM if the pixels(ArrayBuffer) is null, it would generate INVALID_OPERATION if the ArrayBuffer is not large enough. So I think this readPixels API should generate INVALID_ENUM if offset < 0, and generate INVALID_OPERATION if the remainder of the PIXEL_PACK buffer is not large enough. Then these two readPixels APIs are aligned with each other to generate appropriate errors in similar situations.